docs/concepts/move.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Move"
+linkTitle: "Move"
+weight: 1
+---
+
+## Moving Ondat volume instances
+
+A volume instance (replica or primary) can be moved freely between nodes, this can be extremely useful in very specific scenarios, and it's available through a new command in the command-line tool.
```

Comment:
Again, is "volume instance" the terminology we use else where in the docs? Or do we normally say deployment? I'm not sure. Might be worth checking.

Reply:
~A volume instance (replica or primary) can be moved freely between nodes, this can be extremely useful in very specific scenarios, and it's available through a new command in the command-line tool.~
A volume instance (replica or primary) can be moved freely between nodes. This can be extremely useful in ~very~ specific scenarios, and it's available through a new command in the command-line tool.

Reply:
Done